resource/policy.go

```diff
@@ -45,3 +45,11 @@ func ChangesForPolicyUpdate(workload Workload, update policy.Update) (map[string
 	}
 	return result, nil
 }
+
+type Updates map[ID]Update
+
+type Update struct {
```

Comment:
This is a weird place to put this type. At the least, I'd expect it to be called `PolicyUpdate`. But it probably belongs in another package, rather than here. As `update.Policy`, if that doesn't create more problems, for example.